gr-digital/lib/constellation.cc

```diff
@@ -377,6 +402,8 @@ void constellation::max_min_axes()
         d_re_min = d_im_min;
     if (d_re_max == 0)
         d_re_max = d_im_max;
+    
+    d_maxamp = std::max(std::max(d_im_max, d_re_max),std::abs(std::min(d_im_min, d_re_min)));
```

Comment:
I find the following clearer to read:
```suggestion
    d_maxamp = std::max({d_re_max, -d_re_min, d_im_max, -d_im_min});
```